ripe/atlas/tools/settings/__init__.py

```diff
@@ -241,4 +241,31 @@ def write(config):
             rc.write(payload)
 
 
+class Aliases(Configuration):
+    """
+    A singleton class to manage user aliases
+    """
+
```

Comment:
Yeah I don't think the word *"singleton"* is exactly correct here as you do instantiate the class in order to use it.  However it's only ever instantiated once so it's sort of a singleton?  It may be the case that originally it was a legit singleton and then we (I?) decided to instantiate it at the bottom for some reason.

From a look at the code though, there doesn't appear to be any real *need* for it to be one though.  You could just as easily define `.get()` as a class method and then do something like:

```python
Configuration.get()
conf = Configuration  # you could even just skip this and fix the imports
```

But convention (as you find in libraries like requests I believe) appears to go the route of instantiating the class and then just let people import it as we're doing.

In the case of the `Aliases` class, it only has one (static) method, so it doesn't need to be instantiated either.